backend/lib/components/Auth0.js

```diff
@@ -1,47 +1,69 @@
 const axios = require("axios");
-const {
-  config: { auth },
-} = require("../../config");
-const User = require("./User");
+const qs = require("querystring");
+const { config } = require("../../config");
 
-const oauth = async () => {
-  return axios({
-    method: "POST",
-    url: `${auth.host}/oauth/token`,
-    data: {
-      client_id: auth.clientId,
-      client_secret: auth.secretKey,
-      audience: `${auth.host}/api/v2/`,
-      grant_type: "client_credentials",
-    },
-  })
+const errorHandler = (err) => {
+  const { response } = err;
+  const { error, message } = response.data;
+  const statusCode = response.status || response.data.statusCode;
+  // eslint-disable-next-line no-throw-literal
+  throw { statusCode, error, message };
+};
+
+const buildOauthUrl = (provider) => {
+  const qParams = [
```

Comment:
It's probably better to use `querystring.stringify` to pass in an object of query parameters; see https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options

Reply:
Good point! Will do